.wp-env.json

```diff
@@ -1,6 +1,6 @@
 {
 	"core": "WordPress/WordPress",
-	"plugins": [ "." ],
+	"plugins": [ ".", "./plugins/time-to-read-block" ],
```

Comment:
If I'm not wrong, this means the plugin will be enabled when running things like performance tests and e2e tests for Gutenberg. While it may be ok for e2e tests, I think we shouldn't enable these plugins in performance tests.

Reply:
Good catch, I think you're right. I'll try a mapping instead, like we do for the test plugins.